src/components/Dialog/ModelessDialog.tsx

```diff
@@ -284,7 +299,39 @@ const Title = styled.div<{ themes: Theme }>`
     margin: ${spacingByChar(1)} ${spacingByChar(1)} ${spacingByChar(1)} 0;
   `}
 `
+
+const DialogHandler = styled.div<{ themes: Theme }>`
+  ${({ themes: { color, shadow, spacing, radius } }) => css`
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    margin: auto;
+    padding: ${spacing.X3S};
+    border-radius: ${radius.s} ${radius.s} 0 0;
```

Comment:
nits: `bottom-right` と `bottom-left` を省略記法で明示する必要がなければ展開したさ。
```suggestion
    border-top-left-radius: ${radius.s};
    border-top-right-radius: ${radius.s};
```